tracing/jaeger/src/main/java/io/helidon/tracing/jaeger/JaegerTracerBuilder.java

```diff
@@ -419,6 +421,11 @@ public JaegerTracerBuilder addPropagation(PropagationFormat propagationFormat) {
     public Tracer build() {
         Tracer result;
 
+        if (HelidonOpenTelemetry.AgentDetector.isAgentPresent(config)){
+            return HelidonOpenTelemetry.create(GlobalOpenTelemetry.get(),
+                    GlobalOpenTelemetry.getTracer(this.serviceName), tags);
```

Comment:
I think this formatting is not correct.
It should look like this:
```
if (HelidonOpenTelemetry.AgentDetector.isAgentPresent(config)) {
    return HelidonOpenTelemetry.create(GlobalOpenTelemetry.get(),
                                       GlobalOpenTelemetry.getTracer(this.serviceName), 
                                       tags);
}
```
Every method paramter should be on the new line. It improves readablity.

Reply:
ok...
(https://github.com/helidon-io/helidon/pull/6537#discussion_r1185065282)

Reply:
it was wrong at that point as well. since it was at the level of `return` instead of `create` method

Reply:
fixed :)